src/components/resources-section/resource-sidebar/ResourceSidebar.js

```diff
@@ -32,7 +32,7 @@ const ResourceSidebar = ({
         <Average
           average={parseFloat(activeResource.average_evaluation).toFixed(1)}
         />
-        <LinkButton url={activeResource.url} />
+        <LinkButton url={activeResource.url} label={'Ir a recurso'} />
```

Comment:
keys { } not necessary, you can use "ir a recurso" directly

Reply:
Done